tests/dashboards/verifyExperimentalPostgresqlDashboards_test.js

```diff
@@ -0,0 +1,31 @@
+const assert = require('assert');
+
+Feature('Test PostgreSQL Experimental Dashboards');
+
+Before(async ({ I }) => {
+  await I.Authorize();
+});
+
+Scenario(
+  'PMM-T1365 Verify PostgreSQL Vacuum monitoring dashboard @nightly @dashboards @tempTest',
+  async ({
+    I, experimentalPostgresqlDashboardsPage,
+  }) => {
+    await I.amOnPage(experimentalPostgresqlDashboardsPage.vacuumDashboardPostgres.url);
+    await I.waitForVisible(experimentalPostgresqlDashboardsPage.elements.barValue, 60);
+    const values = await I.grabTextFromAll(experimentalPostgresqlDashboardsPage.elements.barValue);
+
+    values.forEach((value) => {
+      const valueInt = parseInt(value.replace('%', ''), 10);
+
+      assert.ok(valueInt > 0, 'The value for Postgres vacuum is zero, it supposed to be > 0');
```

Comment:
please use "codecept chai" assertions - it has way better logs and reporting, see online doc: https://www.npmjs.com/package/codeceptjs-chai also readme is available in node_modules/codeceptjs-chai
ex: `I.assertAbove(2, 1, 'Target data not above the given value');` should be great here

Reply:
Good idea, done.